### DISCUSSION TITLE
I create an island and no longer do teleportation because of SAFE BLOCK.

### DISCUSSION THREAD

**Discussion by Gr1ZzLy23** [BG-Software-LLC/SuperiorSkyblock2#1181] [Pending Feature Requests] [answered]
Hello, in short this plugin has one unresolved issue.. I am setting up a Skyblock setup for my server, and when I create an island I usually do teleportation to my island, but when I go back to spawn and then try to teleport to my island again, I get this message and why? **(I do all this with a player account, and the player rank has all the permissions listed on your site in the "Player Commands" section.)**

Message: **Island | It seems like this island has no safe blocks. Please contact a staff member.**

But I do not make any changes on the island, I do not even break the block and I do not even stand, but it still does not do teleportation, and what can I do ??

My server is running on Paper 1.12.2.

**Comment by OmerBenGera** [comment, marked as answer]
Already been reported: #1139 

Next time, check for reported bugs before opening new reports. Also, use the Issues tab and not the discussion tab.
